src/panels/lovelace/entity-rows/hui-cover-entity-row.ts

```diff
@@ -0,0 +1,76 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../components/hui-generic-entity-row.js";
+import "../../../components/ha-cover-controls.js";
+import "../../../components/ha-cover-tilt-controls.js";
+
+import { isTiltOnly } from "../../../util/cover-model.js";
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+
+class HuiCoverEntityRow extends LitElement implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: EntityConfig;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      _config: {},
+    };
+  }
+
+  public setConfig(config: EntityConfig): void {
+    if (!config || !config.entity) {
+      throw new Error("Entity not configured.");
+    }
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass || !this.hass.states[this._config.entity]) {

```

Comment:
This is not good. We need to render `<hui-generic-entity-row>` if the entity doesn't exist.

Reply:
Or we need to make a new component that will render our yellow block instead of outsourcing that to `<hui-generic-entity-row>`

Reply:
since we still need to use `hui-generic-entity-row` as that is the label to the row, I'm going to be committing my proposal for use instead of an extracted error row.